x-pack/plugins/reporting/server/lib/layouts/preserve_layout.ts

```diff
@@ -49,14 +49,6 @@ export class PreserveLayout extends Layout implements LayoutInstance {
     return ZOOM;
   }
 
-  public getViewport() {
```

Comment:
Previously, these `getViewport` methods on the Layout types helped make sure the viewport was large enough to show the visualization. Although, I'm not sure if it was needed since the clip option would capture the image even if it was not fully in the viewport.

Now, these methods are no longer used since the window has to be sized to include the entire clip area. This doesn't use the Layout type, instead it uses the `ElementsPositionAndAttribute` data of each capture item in `getScreenshots.ts`